component/main.jsonnet

```
@@ -17,9 +17,11 @@ local admin_secret = kube.Secret(params.admin.secretname) {
   },
 };
 
```

Merge jdbcParams in Jsonnet with something like:

```suggestion
local jdbcParams =
  std.join('&', [
    params.database.jdbcParams,
    if params.database.tls.enabled then
      params.database.tls.jdbcParams
    else
      ''
  ]);
```

---

I decided to move `params.database.jdbcParams` 1 up and default to use the ssl parameters since by default we enable TLS